dm-frontend/dashboard.html

```diff
@@ -36,7 +36,7 @@
                         </div>
 
 
-                        <!--<div class="material-icons mdl-badge mdl-badge--overlap" data-badge="4">notifications</div>-->
+                        <div class="material-icons mdl-badge mdl-badge--overlap" data-badge="4">notifications</div>
```

Comment:
Is this change required now that this feature is to be implemented ?

Reply:
Yes, well it was excluded before so I'm including it for the same reason.